tests/test_load.py

```diff
@@ -338,3 +339,45 @@ def _test_date_range(self, ba):
         for dp in data:
             self.assertEqual(dp['market'], self.MARKET_CHOICES.fivemin)
             self.assertEqual(dp['freq'], self.FREQUENCY_CHOICES.fivemin)
+
+
+class TestEULoad(TestBaseLoad):
+    def setUp(self):
+        super(TestEULoad, self).setUp()
+        self.BA_CHOICES = [i['Code'] for i in control_areas]
```

Comment:
EUClient supports many TSO's (we generally call them BA's, but in EU, BA means something else).